<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>大道至简 - Qt串口小助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QComboBox" name="databitsComboBox"/>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>输入格式</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QComboBox" name="outputFormatComboBox"/>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="controlComboBox"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>流控位</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">QLabel {
        color:&quot;blue&quot;
}</string>
      </property>
      <property name="text">
       <string>输出设置</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="14">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;更多调试功能请下载使用“瑞士军刀”：&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#d69545;&quot;&gt;https://gitee.com/qsak/QtSwissArmyKnife/releases&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#d69545;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;获取更多QQ开发资料，请加入Qt开发交流群（QQ群）：&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#d69545;&quot;&gt;723516989&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel {
        color:&quot;blue&quot;
}</string>
      </property>
      <property name="text">
       <string>输入设置</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="devicesComboBox"/>
    </item>
    <item row="16" column="2" rowspan="4">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel {
        color:&quot;blue&quot;
}</string>
      </property>
      <property name="text">
       <string>数据输出</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QComboBox" name="inputFormatComboBox"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="baudrateComboBox"/>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="refreshPushButton">
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>输出格式</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="parityComboBox"/>
    </item>
    <item row="15" column="2">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">QLabel {
        color:&quot;blue&quot;
}</string>
      </property>
      <property name="text">
       <string>输出输入</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="openPushButton">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>校验位</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QPushButton" name="sendPushButton">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="stopBitsComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>设备选择</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel {
        color:&quot;blue&quot;
}</string>
      </property>
      <property name="text">
       <string>设备控制</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QPushButton" name="clearPushButton">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>定时发送</string>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>单位为毫秒，小于10毫秒按10毫秒处理</string>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>devicesComboBox</tabstop>
  <tabstop>baudrateComboBox</tabstop>
  <tabstop>databitsComboBox</tabstop>
  <tabstop>stopBitsComboBox</tabstop>
  <tabstop>parityComboBox</tabstop>
  <tabstop>controlComboBox</tabstop>
  <tabstop>refreshPushButton</tabstop>
  <tabstop>openPushButton</tabstop>
  <tabstop>outputFormatComboBox</tabstop>
  <tabstop>clearPushButton</tabstop>
  <tabstop>inputFormatComboBox</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>sendPushButton</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
